polaris-tokens/src/token-groups/color.ts

```diff
@@ -800,7 +812,7 @@ export const color: {
   },
   'color-text-info-strong': {
     value: colors.teal[900],
-    valueExperimental: colorsExperimental.azure[14],
+    valueExperimental: colorsExperimental.azure[16],
```

Comment:
The [Banner Figma](https://www.figma.com/file/jLLkmo9r28hiqPvf4s90E4/Polaris-Components-TESTING?type=design&node-id=20%3A21&t=0DtfsDGV6aOiQ0LC-1) has `color-text-info-strong` as `azure[16]`